QRCoder/PayloadGenerator.cs

```diff
@@ -15,7 +15,7 @@ public static class PayloadGenerator
         public abstract class Payload
         {
             public virtual int Version { get { return -1; } }
-            public virtual QRCodeGenerator.ECCLevel EccLevel { get { return QRCodeGenerator.ECCLevel.M; } }
+            public virtual QRCodeGenerator.ECCLevel EccLevel { get { return QRCodeGenerator.ECCLevel.Default; } } // null is default
```

Comment:
```suggestion
            public virtual QRCodeGenerator.ECCLevel EccLevel { get { return QRCodeGenerator.ECCLevel.Default; } }
```
As default is -1 now, we don't need the comment any longer. (Mentioning that it is -1 isn't necessary as it's interna of the Enum.)